src/prasad/Bankingsystem.java

```diff
@@ -0,0 +1,70 @@
+package prasad;
+
+public class Bankingsystem {
+	
+	int accountbalance ;
+	int debitamount;
+	int creditamount;
+	int debitcount;
+	int creditcount;
+	int printbalance;
+	static int totaldebitcount;
+	static int totalcreditcount;
+	static int totalprintbalancecount;
+	
+	void creditamt(int amt) {
+		creditamount= creditamount+ amt;
+		accountbalance = accountbalance+amt;
+		creditcount++;
+		totalcreditcount++;
+	}
+	
+	void debitamt(int amt) {
+		debitamount=debitamount+amt;
+		accountbalance = accountbalance-amt;
```

Comment:
what if accountbalance is less than amt ?
you should add if condition.

Reply:
Condition added. Pushing updated code.

Thanks for your comments!